aarch64.dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM balenalib/aarch64-debian-node:10.15-jessie-build
+
+RUN ["cross-build-start"]
+
+RUN apt-get update && apt-get -y install build-essential linux-headers-generic gcc g++ ccache git make
```

Comment:
You should probably add ``install --no-install-recommends -qy``.

Reply:
Building via CI, not needed.